parrots.json

```diff
@@ -182,5 +182,29 @@
     {
       "gif": "parrotbeer.gif",
       "name": "Happy Hour Parrot"
+    },
+    {
+      "gif": "darkbeerparrot",
+      "name": "Parrot with a dark beer"
+    },
+    {
+      "gif": "blondesassyparrot",
+      "name": "Sassy Parrot with a Blonde wig"
+    },
+    {
+      "gif": "bluecluesparrot",
+      "name": "Steve from Blue's Clues Parrot"
```

Comment:
Is there some meme with Steve I don't know about, or do you just enjoy Blues Clues? 😄